packages/runner-sdk/lib/action.ts

```diff
@@ -106,15 +106,6 @@ export abstract class NangoActionBase {
         }
     }
 
-    protected stringify(): string {
-        return JSON.stringify(this, (key, value) => {
-            if (key === 'secretKey') {
-                return '********';
-            }
-            return value;
-        });
-    }
-
```

Comment:
not necessary anymore. Was used by the persistApi related code in `NangoSync` runner-sdk class. 
PersistApi related code now lives in its own file and the secret is not logged anymore